zerver/openapi/zulip.yaml

```diff
@@ -8459,12 +8459,21 @@ paths:
                           Present if `realm` is present in `fetch_event_types`.
 
                           The policy for which users can create bot users in this organization.
-                      realm_create_stream_policy:
+                      realm_create_public_stream_policy:
                         type: integer
                         description: |
                           Present if `realm` is present in `fetch_event_types`.
 
-                          The policy for which users can create streams in this organization.
+                          The policy for which users can create public streams in this organization.
```

Comment:
This should have the same `Changes` entry that was present for `realm_create_private_stream_policy`.